src/Http/Middleware/CsrfProtectionMiddleware.php

```diff
@@ -257,7 +273,10 @@ public function createToken(): string
      */
     public function saltToken(string $token): string
     {
-        $decoded = base64_decode($token);
+        if ($this->isUnsaltedToken($token)) {
```

Comment:
Why do we need to check here before salting? It doesn't read correctly.

Reply:
If the token is hexadecimal only then it shouldn't be salted as it doesn't contain base64 data. I could change the method name to be `isBase64Encoded()` instead.

Reply:
I see. yea, changing the name seems necessary then so it doesn't read like a bug. I guess I'd expect this to be outside of the saltToken() function, but as long as it's consistent.

Reply:
Changed the name to `isHexadecimalToken()` to be less confusing.